library/src/main/java/com/devbrackets/android/exomedia/EMVideoView.java

```diff
@@ -82,7 +84,7 @@
     private ImageView previewImageView;
 
     private TouchVideoView videoView;
-    private VideoSurfaceView exoVideoSurfaceView;
+    private VideoTextureView mExoVideoTextureView;
```

Comment:
Ah sorry, I saw the "mux" stuff and I think accidentally assumed that was your code style. Will fix